microsoft-edge/dualengine/concepts/launching-internet-explorer.md

```diff
@@ -0,0 +1,87 @@
+---
+title: Launching Internet Explorer
+description: Launching Internet Explorer for DualEngine API Use
+author: MSEdgeTeam
+ms.author: edgededev
+ms.topic: conceptual
+ms.prod: microsoft-edge
+ms.localizationpriority: high
+ms.technology: windows-integration
+ms.date: 11/06/2023
+---
+
+# Launching Internet Explorer
+Once you have built your adapter DLL you must launch Internet Explorer such that it is aware that it is being used

```

Comment:
```suggestion
After you have built your adapter DLL, you must launch Internet Explorer such that it is aware that it is being used
```

Reply:
After you've built your adapter DLL, you must launch Internet Explorer such that it is aware that it is being used for the DualEngine API, as follows.